GherkinSyncTool/Synchronizers/TestRailSynchronizer/Model/FeatureFileFolder.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace GherkinSyncTool.Synchronizers.TestRailSynchronizer.Model
+{
+    public class FeatureFileFolder
+    {
+        public string Name { get; set; }
+
+        public string Path { get; set; }
+
+        public string ParentPath { get; set; }
```

Comment:
And not exactly clear what ParentPath means. Parent folder path?

Reply:
Renamed